readme.md

```diff
@@ -1705,43 +1705,44 @@ Additionaly, the errors may have `request` (Got Stream) and `response` (Got Resp
 
 #### got.RequestError
 
-When a request fails. Contains a `code` property with error class code, like `ECONNREFUSED`. All the errors below inherit this one.
+When a request fails. Contains a `code` property with error class code, like `ECONNREFUSED`. If there is no specific code supplied `code` defaults to `ERR_GEN_REQ_ERROR` All the errors below inherit this one.
```

Comment:
I think that `ERR_GOT_REQUEST_ERROR` would suit this a bit better. Wdyt?

Reply:
I like it. I didn't really love using 'general' error and this is more verbose.